File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/SqlExecResVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.util.List;
import java.util.Map;
import java.util.Set;

@Data
public class SqlExecResVo {
    private Set<String> columns;
    private Map<String, Column> tableColumns;

    private Integer queryKey;

    private Integer page;
    private Integer pageSize;

    private Boolean showPagination;
    private String tableName;
    private List<String> primaryKeys;

    private TableData tableData;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/SqlExecResVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen


@AllOpen
class SqlExecResVo {
    var columns: Set<String>? = null
    var tableColumns: Map<String, Column>? = null

    var queryKey: Int? = null

    var page: Int? = null
    var pageSize: Int? = null

    var showPagination: Boolean? = null
    var tableName: String? = null
    var primaryKeys: List<String>? = null

    var tableData: TableData? = null
}
